core/arch/arm/plat-sam/conf.mk

```diff
@@ -10,10 +10,13 @@ $(call force,CFG_PL310,y)
 $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_AT91_MATRIX,y)
 
-CFG_TZDRAM_START ?= 0x30000000
-CFG_TZDRAM_SIZE  ?= 0x800000
+# These values are forced because of matrix configuration for secure area.
+# When modifying these, always update the matrix settings in
+# matrix_configure_slave_h64mx function.
```

Comment:
Nit: by convention we always add parentheses after function names: `matrix_configure_slave_h64mx()`

Reply:
Acked, I fixed it.